deploy/azure/ARM-for-cloud-connectors-identity.json

```diff
@@ -0,0 +1,139 @@
+    "variables": {
+        "userAssignedIdentityName": "[concat('ElasticCloudIdentity-', parameters('elasticCloudConnectorId'))]",
+        "federatedIdentityCredentialsName": "[concat(variables('userAssignedIdentityName'), '/', 'fic-', uniqueString(resourceGroup().id))]",
+        "customRoleDefinition": "[guid('ElasticCloudRoleDefinition-', parameters('elasticCloudConnectorId'))]",
+        "customRoleDefinitionName": "[concat('Elastic Cloud Connector Role - ', parameters('elasticCloudConnectorId'))]",
+        "customRoleAssignment": "[guid('ElasticCloudRoleAssignment-', parameters('elasticCloudConnectorId'))]",
+        "securityReaderRoleAssignment": "[guid('ElasticCloudSecurityReaderRoleAssignment-', parameters('elasticCloudConnectorId'))]"
+    },
+    "parameters": {
+        "elasticStackId": {
+            "type": "string",
+            "minLength": 1,
+            "metadata": {
+                "description": "Subject of the Federated Identity Credential token"
+            }
+        },
+        "elasticCloudConnectorId": {
+            "type": "string",
+            "minLength": 1,
+            "metadata": {
+                "description": "Elastic Cloud Connector unique ID"
+            }
+        },
+        "elasticCloudIssuer": {
+            "type": "string",
+            "defaultValue": "https://evgboidc-dpf8dmdjhnakhpgv.a02.azurefd.net/oidc/",
+            "metadata": {
+                "description": "Issuer for the Elastic Cloud Federated Identity Credential"
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources"
+            }
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.ManagedIdentity/userAssignedIdentities",
+            "apiVersion": "2023-01-31",
+            "name": "[variables('userAssignedIdentityName')]",
+            "location": "[parameters('location')]",
+            "tags": {
+                "elastic-stack-id": "[parameters('elasticStackId')]",
+                "elastic-cloud-connector-id": "[parameters('elasticCloudConnectorId')]"
+            },
+            "resources": [
+                {
+                    "type": "Microsoft.ManagedIdentity/userAssignedIdentities/federatedIdentityCredentials",
+                    "apiVersion": "2023-01-31",
+                    "name": "[variables('federatedIdentityCredentialsName')]",
+                    "dependsOn": [
+                        "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', variables('userAssignedIdentityName'))]"
+                    ],
+                    "properties": {
+                        "issuer": "[parameters('elasticCloudIssuer')]",
+                        "subject": "[parameters('elasticStackId')]",
+                        "audiences": [
+                            "api://AzureADTokenExchange"
```

Comment:
This is the default Audience, we should discuss if we want to stick using that or maybe utilize some custom Audiences if we'd like.